sstables/storage.cc

```diff
@@ -517,7 +518,8 @@ class s3_storage : public sstables::storage {
 
 public:
     s3_storage(shared_ptr<s3::client> client, sstring bucket, sstring dir)
-        : _client(std::move(client))
+        : storage(dir)
+        , _client(std::move(client))
         , _bucket(std::move(bucket))
         , _location(std::move(dir))
```

Comment:
Isn't this now duplicating `_base_dir`?

Reply:
good point, I appears you're right and we just use `_base_dir` instead of `_location`.

Reply:
fixed in v2